README.md

```diff
@@ -32,20 +46,24 @@ Becareful, some commands as no effect on iOS because Apple don't let us to do wh
 
 :warning:(5): Wifi API changes in Android SDK >= 29, restricts certain behaviour:
-  * a. Enable/Disable Wifi Module is deprecated and will always fail (except DO, PO and system apps) [[docs](https://developer.android.com/reference/android/net/wifi/WifiManager#setWifiEnabled(boolean))].
+  * a. Enable/Disable Wifi Module is deprecated and will always fail [[docs](https://developer.android.com/reference/android/net/wifi/WifiManager#setWifiEnabled(boolean))]. If you  want to open "Wifi Setting" in that case then, set the `shouldOpenSettings: true` when calling `setEnabled`.
   * b. For Connecting to Wifi, WEP security is deprecated and will always fail, also the network will be disconnected when the app is closed (if permanent network is required(Check :warning:(5c)), use "Register Network" feature) [[docs](https://developer.android.com/guide/topics/connectivity/wifi-bootstrap))]. By default the connection would not have internet access, to connect to network with internet user `withInternet` which is a different API underneath (this API will not disconnect to network after app closes) [[docs](https://developer.android.com/guide/topics/connectivity/wifi-suggest)].
   * c. Registering Wifi Network, will require user approval - and the network saved would not be controlled by the app (for deletion, updation, etc) [[docs](https://developer.android.com/guide/topics/connectivity/wifi-save-network-passpoint-config)];
 
 ## Access Point
 |                                       Description                                     |      Android       |         iOS          |
 | :------------------------------------------------------------------------------------ | :----------------: | :------------------: |
-| Getting the status of the Access Point (Disable, disabling, enable, enabling, failed) | :white_check_mark: |  :x:  |
-| Enabling / Disabling Access Point                                                     | :white_check_mark: |  :x:  |
-| Getting / Setting new credentials (SSID / Password)                                   | :white_check_mark: |  :x:  |
-| Enabling / Disabling the visibility of the SSID Access Point                          | :white_check_mark: |  :x:  |
+| Getting the status of the Access Point (Disable, disabling, enable, enabling, failed) | :warning:(1a) |  :x:  |
+| Enabling / Disabling Access Point                                                     | :white_check_mark:(1b) |  :x:  |
+| Getting / Setting new credentials (SSID / Password)                                   | :warning:(1a) |  :x:  |
+| Enabling / Disabling the visibility of the SSID Access Point                          | :warning:(1a) |  :x:  |
 | Getting the clients list (IP, BSSID, Device, Reachable)                               | :white_check_mark: |  :x:  |
 | Handling the MAC filtering                                                            | :sos: |  :x:  |
```

Comment:
@mavyfaby Is this deprecated or is working?

Reply:
Seems like getting status of Access point on Android 10 works for me. I have only Android 10 and 5 devices. So I can't test on other android versions.

Reply:
I am asking about "Getting clients list"

Reply:
Yes, it does work on Android 10 too.

Reply:
Doesn't work on android 10 for me, the error says:

```
E/class info.whitebyte.hotspotmanager.WifiApManager$1(11535): java.io.FileNotFoundException: /proc/net/arp: open failed: EACCES (Permission denied)
```